firmware/examples/PWM/source/main.cpp

```diff
@@ -0,0 +1,37 @@
+#include <cstdarg>
+#include <cstdio>
+#include <cstdint>
+#include "L0_LowLevel/delay.hpp"
+#include "L1_Drivers/gpio.hpp"
+#include "L1_Drivers/pwm.hpp"
+#include "L2_Utilities/debug_print.hpp"
+
+int main(void)
+{
+    DEBUG_PRINT("Pwm Application Starting...\n");
```

Comment:
Debug print already adds a newline character to the end of your string. The \n isn't needed here.